### ISSUE TITLE
Support OpenShift/Kubernetes API

### ISSUE THREAD

**Issue by ibotty** [redhat-cop/keepalived-operator#88]
It would be great to allow the operator to serve the internal and external Kubernetes/OpenShift API.

**Comment by raffaelespazzoli** [comment]
there is a chicken egg situation there that cannot be worked around. But if you are ok with creating a secondary loadbalancer for the master api, then it is possible.

**Comment by ibotty** [comment]
Yes, I know.  That's the hard part.  For now I use (a fork of) kube-karp as static pods on the control plane for the API.  It would be nice to have a solution that works for all load balancing needs though.  I assume the operator would have to sync the necessary resources to the control plane's file system (`/etc/kubernetes/{manifests,static-pod-resources}`).  Would this extra complexity be accepted?
